core/src/test/java/io/confluent/kafka/schemaregistry/rest/RestApiTest.java

```diff
@@ -484,17 +486,243 @@ public void testCanonicalization() throws Exception {
     String schema = "{   \"type\":   \"string\"}";
     String subject = "test";
     assertEquals("Registering a new schema should succeed",
-            1,
-            restApp.restClient.registerSchema(schema, subject));
+                 1,
+                 restApp.restClient.registerSchema(schema, subject));
 
     assertEquals("Registering the same schema should get back the same id",
-            1,
-            restApp.restClient.registerSchema(schema, subject));
+                 1,
+                 restApp.restClient.registerSchema(schema, subject));
 
     assertEquals("Lookup the same schema should get back the same id",
-            1,
-            restApp.restClient.lookUpSubjectVersion(schema, subject)
-                    .getId().intValue());
+                 1,
+                 restApp.restClient.lookUpSubjectVersion(schema, subject)
+                     .getId().intValue());
+  }
+
+  @Test
+  public void testDeleteSchemaVersionBasic() throws Exception {
+    List<String> schemas = TestUtils.getRandomCanonicalAvroString(2);
+    String subject = "test";
+
+    TestUtils.registerAndVerifySchema(restApp.restClient, schemas.get(0), 1, subject);
+    TestUtils.registerAndVerifySchema(restApp.restClient, schemas.get(1), 2, subject);
+
+    assertEquals("Deleting Schema Version Success", (Integer) 2, restApp.restClient
+        .deleteSchemaVersion
+            (subject, "2"));
+
+    List expectedVersionList = new ArrayList();
+    expectedVersionList.add(1);
```

Comment:
These can be inlined as, e.g., `Collections.singletonList(1)` if you want to tighten this up a bit.